src/openssl.c

```diff
@@ -7173,6 +7173,14 @@ static int xs_add(lua_State *L) {
 				X509_CRL_free(crl_dup);
 				return auxL_error(L, auxL_EOPENSSL, "x509.store:add");
 			}
+
+			/* When a CRL was added, enable CRL checking (entire chain) */
+			X509_VERIFY_PARAM *param = X509_VERIFY_PARAM_new();
+			X509_VERIFY_PARAM_set_flags(param, X509_V_FLAG_CRL_CHECK);
+			X509_VERIFY_PARAM_set_flags(param, X509_V_FLAG_CRL_CHECK_ALL);
+			X509_STORE_set1_param(store, param);
```

Comment:
I'm hesistant to do this by default when openssl itself doesn't do it by default.
Instead, could you add bindings to `X509_VERIFY_PARAM_set_flags` so that lua users can do this themselves?